sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/OpenTelemetry.Exporter.AzureMonitor.csproj

```diff
@@ -9,7 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="System.Text.Json" />
-    <PackageReference Include="OpenTelemetry" Version="0.6.0-beta.1" IsImplicitlyDefined="true" />
```

Comment:
@rajkumar-rangaraj I moved the version to the Packages.Data.props.
I don't tell if the `IsImplicitlyDefined` is required.

This is the only doc page I could find: https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio#update-a-package

Please advise.

Reply:
CI has failed without `IsImplicitlyDefined` hence added it. Without this attribute if CI could pass then we don't need to worry about it.